### PR TITLE
My Contribution to Create your own adventure

### PR DESCRIPTION
**Pull request by krushed18**

### CODE REVIEW

**Comment by krushed18** [comment]
Please consider my contribution, @casey-collab

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: english/swami-vivekananda.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+﻿Want to know some of Swami Vivekananda's teachings? [Refer this](./teachings-of-swami-vivekananda/quotes.md)
```
Comment (casey-collab) [inline english/swami-vivekananda.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

File: hindi/swami-vivekananda.md [hunk 2]
```diff
@@ -0,0 +1 @@
+﻿पता है स्वामी विवेकानंद जी क्या कहते थे? [यहाँ देखो|](./teachings-of-swami-vivekananda/quotes.md)
```
Comment (casey-collab) [inline hindi/swami-vivekananda.md]: This file has also been added to a directory that already contained a page.

Comment (casey-collab) [inline hindi/swami-vivekananda.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

File: english/swami-vivekananda.md (hunk 1 above)
Comment (casey-collab) [inline english/swami-vivekananda.md]: This page is also unreachable.

**Comment by casey-collab** [comment]
It looks like the files `english/teachings-of-swami-vivekananda/quotes.md` and `hindi/teachings-of-swami-vivekananda/quotes.md` are empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to each file, or delete the options that lead to them.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**